docs/docs/12-context.md

```diff
@@ -64,7 +64,7 @@ var Button = React.createClass({
   },
   render: function() {
     return (
-      <button style={{'{{'}}background: this.context.color}}>
+      <button style={{background: this.context.color}}>
```

Comment:
This was correct as-is, as per our MD generator.  You should try rendering the docs as described in the README in the docs directory.